.gitignore

```diff
@@ -1,2 +1,6 @@
 *~
 #*#
+.vscode
+creds.py
+/api/__pycache__
+/api/hive13_api/__pycache__
```

Comment:
does this need to be modified somewhat to exclude .pyc too?

Reply:
Should be fixed now. I missed adding a forward slash at the end.